klippy/kinematics/cartesian.py

```diff
@@ -118,7 +118,15 @@ def check_move(self, move):
             self.max_z_velocity * z_ratio, self.max_z_accel * z_ratio)
     def get_status(self, eventtime):
         axes = [a for a, (l, h) in zip("xyz", self.limits) if l <= h]
-        return { 'homed_axes': "".join(axes) }
+        axes_min = {}
+        axes_max = {}
```

Comment:
It would be preferable to export a `homing.Coord()` instead of a dictionary so that it matches the format of other exported coordinates.  For example, see how `toolhead.get_status()` exports the `'position'` parameter.